### DISCUSSION TITLE
Release/7.2 build errors - should I upload results?

### DISCUSSION THREAD

**Discussion by Zenchess** [dolphinsmalltalk/Dolphin#1302] [General]
I got the follwing failures on 7.2 release branch
FAILURES:
DisplayMonitorTest>>testContainingPoint
DisplayMonitorTest>>testNearestPoint

ERRORS:
DialogViewTest>>testRemainsOnScreen

Should I double click the "UploadTestResults.ps1" file?

I suspect the failures are due to my unique monitor setup.  I have 3 monitors, 
the middle is 3440x1440, the left is 1920x1080, and the right one is 1920x1200.

If I can help test let me know!

**Comment by blairmcg** [comment]
Yes, they are probably due to assumptions in the tests that are not always correct.
It would be useful if you just attach the `RegressionTests.xml` file here.
Also what do you see if you evaluate:
```smalltalk
DisplayMonitor allMonitors collect: [:each | each -> each workArea]
```

**Reply by Zenchess** [reply]
#(a DisplayMonitor('\\.\DISPLAY1') -> -1920@362 corner: 0@1402 a DisplayMonitor(primary: '\\.\DISPLAY2') -> 0@0 corner: 3440@1400 a DisplayMonitor('\\.\DISPLAY3') -> 3440@232 corner: 5360@1392)

[RegressionTests.txt](https://github.com/user-attachments/files/16367335/RegressionTests.txt)

xml uploading was not supported so I just renamed it to .txt

**Reply by blairmcg** [reply]
> #(a DisplayMonitor('\.\DISPLAY1') -> -1920@362 corner: 0@1402 a DisplayMonitor(primary: '\.\DISPLAY2') -> 0@0 corner: 3440@1400 a DisplayMonitor('\.\DISPLAY3') -> 3440@232 corner: 5360@1392)
> 
> [RegressionTests.txt](https://github.com/user-attachments/files/16367335/RegressionTests.txt)
> 
> xml uploading was not supported so I just renamed it to .txt

That's useful thank you. I can why they would fail with that configuration. The first two are easily fixed, the DialogViewTest perhaps less so, but I think I can see how to do it.
